tutorials/2.6/system-thread.php

```diff
@@ -222,6 +222,9 @@ classes are a good solution.
     since threads run in parallel, it means that a variable or function might be used concurrently from several threads at the same time.
     If the operation is not <em>thread-safe</em>, it can lead to undefined behavior (ie. it might crash or corrupt data).
 </p>
+<p class="important">
+    SFML objects are usually not thread-safe, you must protect them to use them in multiple threads at the same time !
```

Comment:
```suggestion
    SFML objects are usually not thread-safe. You must protect them to use them in multiple threads at the same time!
```

Reply:
Wow I can directly click in Github to commit your suggestions that's so cool !